### ISSUE TITLE
Automated testing

### ISSUE THREAD

**Issue by eddiejaoude** [EddieHubCommunity/LinkFree#226]
**tools**
- Cypress
- Cucumber

**test steps**
- [ ] home page lists multiple users
- [ ] click on first/random user and check the data display is correct according to their json file

**Comment by rohan-kulkarni-25** [comment]
>     * ser and check the data display is correct according to their json file

Yes this will be required. I have never added any test or never worked with it but will like to work this time. Will you please guide me Eddie

**Comment by BOLT04** [comment]
automated testing 💯 💯 
Quick question: is there a URL these tests would use? I couldn't test the deployed version on  linkfree.eddiehub.org/

**Comment by rohan-kulkarni-25** [comment]
> automated testing 100 100
> Quick question: is there a URL these tests would use? I couldn't test the deployed version on linkfree.eddiehub.org/

The deployed version is running on [LinkFree](https://github.com/EddieHubCommunity/LinkFree) Please have a look !

**Comment by BOLT04** [comment]
I had a look @rohan-kulkarni-25, but the app doesn't show up. Only a list of links to other sites like Netflix...
Also the link isn't with `https`, but even inputting it manually doesn't work.

**Comment by rohan-kulkarni-25** [comment]
> I had a look @rohan-kulkarni-25, but the app doesn't show up. Only a list of links to other sites like Netflix...
> Also the link isn't with `https`, but even inputting it manually doesn't work.

You need to refresh the ads will go

**Comment by WendallHersh** [comment]
do you need to have it in cucumber?

**Comment by WendallHersh** [comment]
https://github.com/EddieHubCommunity/LinkFree/pull/305

**Comment by schmelto** [comment]
@eddiejaoude is this closed by #380 ?

**Comment by krishguptadev** [comment]
I think this is to be closed 🤔
